web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -69,14 +74,18 @@ const HfiCalculatorPage: React.FunctionComponent = () => {
         </Container>
       ) : (
         <Container maxWidth={'xl'}>
+          <FormControl>
+            <DatePicker
+              date={dateOfInterest}
+              onChange={setDateOfInterest}
+              updateDate={updateDate}
+            />
+          </FormControl>
           <DailyViewTable
-            title="HFI Calculator Daily View"
             testId="hfi-calc-daily-table"
             fireCentres={fireCentres}
             dailiesMap={dailiesMap}
-            currentDay={currentDay.toLocaleString()}
-            previousDay={previousDay}
-            nextDay={nextDay}
+            currentDay={dateOfInterest}
```

Comment:
Is this used in `DailyViewTable` anymore?

Reply:
nope! good catch. I've deleted it now